src/app.js

```diff
@@ -25,22 +27,25 @@ if(getEnv() === 'development') {
       chunkModules: false,
       modules: false
     }
-  }));
-  app.use(require('webpack-hot-middleware')(compiler));
+  }))
+  app.use(require('webpack-hot-middleware')(compiler))
 }
 
-app.use(logger('dev'));
-app.use(bodyParser.json());
-app.use(bodyParser.urlencoded({ extended: false }));
-app.use(cookieParser());
+app.use(logger('dev'))
+app.use(bodyParser.json())
+app.use(bodyParser.urlencoded({ extended: false }))
+app.use(cookieParser())
 app.use(express.static(path.join(__dirname, 'public')))
 auth(app)
 
 app.use('/api/task', task)
+app.use('/noob', noob)
```

Comment:
actually, get rid of `admins` from the route name, as it doesn't really have anything to do with `admins`. It is modifying `template_tasks` and it just so happens that an `admin` has permissions to do it. Tomorrow we can create new roles who could potentially create `template_tasks`. So I would recommend removing `admins` from the route name, so the route would look something like this - `/template_tasks/...`